### ISSUE TITLE
pip3 install fails -> NameError: global name 'unicode' is not defined

### ISSUE THREAD

**Issue by jardaroh** [philippbosch/django-werkzeug-debugger-runserver#1]
$ sudo pip3 install django-werkzeug-debugger-runserver

Downloading/unpacking django-werkzeug-debugger-runserver
  Downloading django-werkzeug-debugger-runserver-0.2.tar.gz
  Running setup.py egg_info for package django-werkzeug-debugger-runserver
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
      File "/tmp/pip_build_root/django-werkzeug-debugger-runserver/setup.py", line 12, in <module>
        long_description=readme('README.rst'),
      File "/tmp/pip_build_root/django-werkzeug-debugger-runserver/setup.py", line 6, in readme
        return unicode(file.read())
    NameError: global name 'unicode' is not defined
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 16, in <module>

  File "/tmp/pip_build_root/django-werkzeug-debugger-runserver/setup.py", line 12, in <module>

```
long_description=readme('README.rst'),
```

  File "/tmp/pip_build_root/django-werkzeug-debugger-runserver/setup.py", line 6, in readme

```
return unicode(file.read())
```

NameError: global name 'unicode' is not defined

---

Cleaning up...
Command python setup.py egg_info failed with error code 1 in /tmp/pip_build_root/django-werkzeug-debugger-runserver

**Comment by philippbosch** [comment]
I'm not using Python 3, but could you please check if the latest master works for you? If so, I'd push this to PyPI. Thanks!

**Comment by philippbosch** [comment]
@jardaroh @rightaway

**Comment by philippbosch** [comment]
Fixed in 0.3
